src/pallets/static/style.css

```diff
@@ -8,27 +13,105 @@ header nav a.logo {
 }
 
 header nav a.logo img {
-  height: 3em;
-  max-width: none;
+  width: 3em;
+}
+
+header nav {
+  display: flex;
+  justify-content: space-between;
+  justify-items: center;
+  padding: 0;
+  margin: 0;
+}
+
+header nav ul {
+  padding: 0 1rem;
 }
 
 @media (max-width: 767px) {
   header nav {
-    overflow-x: auto;
+    overflow-x: hidden;
   }
 
   header nav a.logo img {
-    height: 2em;
+    width: 3em;
+  }
+}
+
+.logo {
```

Comment:
What is this changing? The screenshot looks the same, and pico already handles nav layout.

Reply:
This is a spacing change, in this scenario flex is acting like a spreadsheet cell with the alignment set to middle. Giving even space above and below. The rem gap brings the text and image a little closer together. This allows the image and text to behave in more of a responsive manner.